src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -381,14 +381,11 @@ export class CommandDetectionCapability extends Disposable implements ICommandDe
 
 		if (newCommand) {
 			this._commands.push(newCommand);
```

Comment:
can `_commitCommandFinished` be removed as it's no longer scheduled anywhere?

Reply:
Yes, I thought I did this